test/e2e/functional/tests/bfdprofile.go

```diff
@@ -106,3 +111,33 @@ bfd-profiles:
 func uint32Ptr(n uint32) *uint32 {
 	return &n
 }
+
+func createMetalLBResource() {
+	metallb := &metallbv1beta1.MetalLB{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "metallb",
+			Namespace: OperatorNameSpace,
+		},
+	}
+	By("Creating a MetalLB resource")
+	err := testclient.Client.Create(context.Background(), metallb)
+	Expect(err).ToNot(HaveOccurred())
+	configmap := &corev1.ConfigMap{}
+	Eventually(func() (string, error) {
```

Comment:
I believe here you can just check for err == nill in the eventually loop.
The name will match the configmap name, as you are passing it.